src/controllers/userController.js

```diff
@@ -0,0 +1,90 @@
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  const users = userService.getAll();
+
+  if (!users) {
+    res.send(users);
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+  const user = userService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const update = (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+  const user = userService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
+    res.sendStatus(422);
+
+    return;
+  }
+
+  const updatedUser = userService.updateUser({ id: +id, name });
```

Comment:
It would be better to check if the 'name' is not an empty string. Currently, if an empty string is passed, it will pass the check and an empty name user would be created which is not meaningful.